### PR TITLE
hack: add example that has full peer/client tls

### PR DESCRIPTION
**Pull request by polvi**
Three node k8s example with full client/peer tls and script to generate k8s secrets objects required.

### CODE REVIEW

**Comment by yichengq** [comment]
pretty good IMO

File: hack/kubernetes-deploy/README.md [hunk 1]
```diff
@@ -15,7 +17,45 @@ $ # now deploy a service that consumes etcd, such as vulcand
...
+services/etcd2
+```
+
+A secret for clients is created and named `etcd-client-key`. Test it out with etcdctl. 
+
+```
+$ kubectl create -f etcdctl.yml
+$ kubectl exec etcdctl -- /etcdctl \
+-peers https://etcd0:2379,https://etcd1:2379,https://etcd2:2379 \
+-ca-file=/etc/etcd/client-ca/ca.crt \
```
Comment (yichengq) [inline hack/kubernetes-deploy/README.md]: we encourage to use `-trusted-ca-file` and `-enable-client-auth`. The equivalent for this line is `-trusted-ca-file=/etc/etcd/client-ca/ca.crt -enable-client-auth`

**Comment by kelseyhightower** [comment]
@polvi This makes me smile!

File: hack/kubernetes-deploy/README.md (hunk 1 above)
Comment (xiang90) [inline hack/kubernetes-deploy/README.md]: @polvi Can you try to clean this up and then we can merge this in? Thanks!

**Comment by xiang90** [comment]
closing this. users should try etcd operator instead if they want to run etcd atop k8s.

**Pull request closed without merging**